Limit simultaneous install stages to 20

TeddyAndrieux opened the pull request:
**Component**:

'build'

**Context**: 

Do not consume too much CI resources during build.

**Summary**:

Limits the number of simultaneous "install stages" in parallel to 20.

**Acceptance criteria**: 


---

bert-e commented:
# Hello teddyandrieux,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Conflict




There is a conflict between your branch `improvement/limit-simultaneous-install-stages` and the
destination branch `development/2.0`.

Please resolve the conflict on **the feature branch** (`improvement/limit-simultaneous-install-stages`).

```bash
 $ git fetch
 $ git checkout origin/improvement/limit-simultaneous-install-stages
 $ git merge origin/development/2.0
 $ # <intense conflict resolution>
 $ git commit
 $ git push origin HEAD:improvement/limit-simultaneous-install-stages
```

TeddyAndrieux committed:
eve: Rename `multiple-nodes` stage to `multiple-nodes-centos`

Rename `multiple-nodes` stage to `multiple-nodes-centos` to have the
same naming on all development branches to being able to add some
filtering on this stage like `simultaneous_build`.

TeddyAndrieux committed:
eve: Limit simultaneous build for install stage to 20

Limit number of parallel stage that need to spawn Openstack VMs to 20 to
not consume too much ressource during builds.

NOTE: `simultaneous_build` will not fail the stage if we already have 20
running, but will just wait for one to finish before starting the new
one.

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/2.1/improvement/limit-simultaneous-install-stages` with contents from `improvement/limit-simultaneous-install-stages`
and `development/2.1`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/2.1/improvement/limit-simultaneous-install-stages origin/development/2.1
 $ git merge origin/improvement/limit-simultaneous-install-stages
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/2.1/improvement/limit-simultaneous-install-stages
```

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/2.4/improvement/limit-simultaneous-install-stages` with contents from `w/2.3/improvement/limit-simultaneous-install-stages`
and `development/2.4`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/2.4/improvement/limit-simultaneous-install-stages origin/development/2.4
 $ git merge origin/w/2.3/improvement/limit-simultaneous-install-stages
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/2.4/improvement/limit-simultaneous-install-stages
```

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/limit-simultaneous-install-stages` into
`development/2.0`
* [w/2.1/improvement/limit-simultaneous-install-stages](https://github.com/scality/metalk8s/compare/development/2.1...w/2.1/improvement/limit-simultaneous-install-stages) will be merged into `development/2.1` 
* [w/2.2/improvement/limit-simultaneous-install-stages](https://github.com/scality/metalk8s/compare/development/2.2...w/2.2/improvement/limit-simultaneous-install-stages) will be merged into `development/2.2` 
* [w/2.3/improvement/limit-simultaneous-install-stages](https://github.com/scality/metalk8s/compare/development/2.3...w/2.3/improvement/limit-simultaneous-install-stages) will be merged into `development/2.3` 
* [w/2.4/improvement/limit-simultaneous-install-stages](https://github.com/scality/metalk8s/compare/development/2.4...w/2.4/improvement/limit-simultaneous-install-stages) will be merged into `development/2.4` 
* [w/2.5/improvement/limit-simultaneous-install-stages](https://github.com/scality/metalk8s/compare/development/2.5...w/2.5/improvement/limit-simultaneous-install-stages) will be merged into `development/2.5` 
* [w/2.6/improvement/limit-simultaneous-install-stages](https://github.com/scality/metalk8s/compare/development/2.6...w/2.6/improvement/limit-simultaneous-install-stages) will be merged into `development/2.6` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nootal

* @alexandre-allard-scality

* @lucieleonard

* @sayf-eddine-scality

* @slaperche-scality

* @nicolast

* @gdemonet

* @carlito-scality

* @thomasdanan

* @jbertran

* @teddyandrieux

* @cmonfort

* @monpote

TeddyAndrieux commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nootal

* @alexandre-allard-scality

* @lucieleonard

* @sayf-eddine-scality

* @slaperche-scality

* @nicolast

* @gdemonet

* @carlito-scality

* @thomasdanan

* @jbertran

* @teddyandrieux

* @cmonfort

* @monpote










*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.0`

* :heavy_check_mark: `development/2.1`

* :heavy_check_mark: `development/2.2`

* :heavy_check_mark: `development/2.3`

* :heavy_check_mark: `development/2.4`

* :heavy_check_mark: `development/2.5`

* :heavy_check_mark: `development/2.6`



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.0`

* :heavy_check_mark: `development/2.1`

* :heavy_check_mark: `development/2.2`

* :heavy_check_mark: `development/2.3`

* :heavy_check_mark: `development/2.4`

* :heavy_check_mark: `development/2.5`

* :heavy_check_mark: `development/2.6`



The following branches have **NOT** changed:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`



Please check the status of the associated issue None.

Goodbye teddyandrieux.
